.github/.wordlist.txt

```diff
@@ -905,6 +905,7 @@ MoveWithOnOff
 MPSL
 MRP
 MTD
+MTR
```

Comment:
MTR is not a word. Could we spell it out as 'matter' or is this some specific fixed name in `OPTIGA&trade; Trust M MTR provisioning` ?